scs-app/src/app/fulfill_major/fulfill_major.component.scss

```diff
@@ -0,0 +1,31 @@
+#fulfill-major-tool {
+    margin: 5%;
+}
+
+#title {
+    font-size: 2em;
+}
+#blurb {
+    width: 90%;
+}
+
+.flex-container {
+    display: flex;
+    flex-flow: row wrap;
+}
+.flex-item {
```

Comment:
if we need to make customized scss for "flex-item" i think we should make a separate name i.e. typeahead-item { flex-basis: 45% } reason being "flex-item" is a bit confusing code wise as the flex-item css already exists

Reply:
changed it all to bootstrap grid!